.gitignore

```diff
@@ -1,6 +1,7 @@
 env
 __pycache__
 .vscode
+backend/configuration.py
```

Comment:
```suggestion
```

We could use .env files for driver string and leave this file in repo.
https://pypi.org/project/python-dotenv/